src/Algorithms/TMJets/common.jl

```diff
@@ -37,7 +39,21 @@ _initialize(X0::IntervalArithmetic.Interval, orderQ, orderT) = IntervalBox(X0)
 
 # zonotopic sets
 function _initialize(X0::AbstractZonotope, orderQ, orderT)
-    X = overapproximate(X0, TaylorModelReachSet, orderQ=orderQ, orderT=orderT)
+    X0z = convert(Zonotope, X0)
+    if order(X0z) == 1
+        X = overapproximate(X0z, TaylorModelReachSet, orderQ=orderQ, orderT=orderT)
+    else
+        X = overapproximate(X0z, TaylorModelReachSet, orderQ=orderQ, orderT=orderT, box_reduction=true)
+    end
+    return set(X)
+end
+
+function _initialize(X0::CartesianProduct{N, <:AbstractHyperrectangle, <:AbstractHyperrectangle}, orderQ, orderT) where {N}
+    return convert(IntervalBox, convert(Hyperrectangle, X0))
+end
+
+function _initialize(X0::CartesianProduct{N, <:Zonotope, <:Interval}, orderQ, orderT) where {N}
```

Comment:
the motivation for this initial state is when we we have n-1 "state" variables and 1 "control" variable and the zonotope has order 1 as in [this model](https://github.com/JuliaReach/NeuralNetworkAnalysis.jl/pull/172/files#diff-391083c2fe8f6e3567dc7b5c9c1091f2f4b1641815bd49ea24db9349fe019c53R40)

i think in general though, the zonotope will have order at most 2 (first half of generators for the linear part, second half for the remainders which enclose the nonlinear part). we may special case all this inside the `_overapproximate_structure`.  i'll leave that for another PR.